src/EnergyPlus/ScheduleManager.cc

```diff
@@ -1210,6 +1221,9 @@ namespace Sched {
         //  A4 , \field Complex Field #2
         //  A5 , \field Complex Field #3
 
+        // When InitConstantScheduleData is called, TimeStepsInHour is 0, so we delay it here
+        static_cast<DaySchedule *>(s_sched->daySchedules[SchedNum_AlwaysOff])->tsVals.assign(Constant::iHoursInDay * s_glob->TimeStepsInHour, 0.0);
```

Comment:
In ProcessScheduleInput, I reassign the DaySchedule to the right size now that we know the TimeStepsInHour.